modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1783,6 +1783,18 @@ public DefaultCodegen() {
         cliOptions.add(enumUnknownDefaultCaseOpt);
         this.setEnumUnknownDefaultCase(false);
 
+        CliOption removeEnumValuePrefixOpt = CliOption.newBoolean(
+                CodegenConstants.REMOVE_ENUM_VALUE_PREFIX,
+                CodegenConstants.REMOVE_ENUM_VALUE_PREFIX_DESC).defaultValue(Boolean.TRUE.toString());
+        Map<String, String> removeEnumValuePrefixOpts = new HashMap<>();
+        removeEnumValuePrefixOpts.put("false",
+                "No changes to the enum's are made.");
+        removeEnumValuePrefixOpts.put("true",
+                "With this option disabled, each enum will have the common prefix between them removed. This is the default option.");
```

Comment:
Not: disabled -> enabled? Since this is when it's true?

Reply:
I did not understand your question. This option was implemented by someone else and that's how it ended up, I just wrote what happens with each option.

Reply:
I think it's with the option _enabled_, each enum will have the common prefix between them removed.

Reply:
Exactly. It was done like this. But it's been like this since Feb 8, 2020. PR link merged #5166